### PR TITLE
Update ctas and image in /kubeflow/what-is-kubeflow

### LINKED ISSUE #8795: [COPY UPDATE] /what-is-kubeflow
Please update /what-is-kubeflow from the [copy doc](https://docs.google.com/document/d/1m-96RlGkbzFa1KaGAxYFpY0ztIlsTuViCjXSAlErQlg/edit?ts=5fb7b3eb#)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Updated a few links
- Used the new illustration

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/kubeflow/what-is-kubeflow
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to and resolve comments in the [copy doc](https://docs.google.com/document/d/1m-96RlGkbzFa1KaGAxYFpY0ztIlsTuViCjXSAlErQlg/edit?ts=5fb7b3eb#)

## Issue / Card

Fixes #8795

### CODE REVIEW

**Comment by SirSamTumless** [comment]
@pmahnke A button is missing here
Copy doc: Screenshot 2020-11-24 at 09 55 01

Demo: 
Screenshot 2020-11-24 at 09 55 39

**Comment by pmahnke** [comment]
@SirSamTumless that's because he added it _AFTER_ I put in the PR.

File: templates/kubeflow/what-is-kubeflow.html [hunk 1]
```diff
@@ -512,14 +511,18 @@ <h2 class="u-fixed-width u-sv1">Who uses Kubeflow?</h2>
...
-      <a class="p-button--positive" href="/kubeflow/install">
-      Try Kubeflow
-      </a>
+      <p class="u-sv2 p-heading--four">Try-out Kubeflow on your K8s deployment. Or on MicroK8s - Zero-ops Kubernetes with high availability.</p>
+      <p>Single-command deploy locally on your desktop, public cloud VM or on-prem server.</p>
+      <p>
+        <a class="p-button--positive p-link--external" href="https://charmed-kubeflow.io/docs/install">
+        Try Kubeflow
+        </a>
+        <a class="p-link--external" href="https://charmed-kubeflow.io">
```
Comment (bethcollins92) [inline templates/kubeflow/what-is-kubeflow.html:520]: In the doc this link is: 
```suggestion
        <a class="p-link--external" href="https://microk8s.io/docs/addon-kubeflow">
```

**Review by bethcollins92 — APPROVED** [review]
one comment about the link but otherwise LGTM 👍

**Pull request merged**